ServerTaskHelper/AzureFunctionBasicSample/CheckLogger.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Net.Http;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Text;
+
+namespace AzureFunctionBasicSample
+{
+    /// <summary>
+    /// Class <c>CheckLogger</c> logs messages to checks console within pipeline run.
+    /// </summary>
+    public sealed class CheckLogger
+    {
+
+        private CheckLogger() { }
+
+        /// <summary>This is the entry point of checks logging feature.
+        /// <param name="httpClient">HttpClient instance</param>
+        /// <param name="projectId">ProjectId system variable from request headers: system.TeamProjectId</param>
+        /// <param name="authToken">AuthToken system variable from request headers: system.AccessToken</param>
+        /// <param name="hubName">HubName system variable from request headers: system.HostType</param>
+        /// <param name="planId">PlanId system variable from request headers: system.PlanId</param>
+        /// <param name="planUrl">PlanUrl system variable from request headers</param>
+        /// <param name="jobId">JobId system variable from request headers: system.JobId</param>
+        /// <param name="timelineId">TimelineId system variable from request headers: system.TimelineId</param>
+        /// <param name="taskInstanceId">TaskInstanceId system variable from request headers: system.TaskInstanceId</param>
+        /// <param name="message">Message logged to the checks console</param>
+        /// </summary>
+        public static void WriteToCheckConsole(HttpClient httpClient, string planUrl, string projectId, string hubName, string planId, string jobId, string timelineId, string taskInstanceId, string authToken, string message)
+        {
+            var pagesFolder = Path.Combine(Path.GetTempPath(), "pages");
+            Directory.Delete(pagesFolder, true);
+            Directory.CreateDirectory(pagesFolder);
+
+            var logFileName = Path.Combine(pagesFolder, $"taskLog.log");
+            var pageData = new FileStream(logFileName, FileMode.CreateNew);
+            var pageWriter = new StreamWriter(pageData, Encoding.UTF8);
+
+            SendTaskLogFeeds(httpClient, authToken, planUrl, projectId, hubName, planId, jobId, timelineId, message);
+            pageWriter.WriteLine(message);
+            pageWriter.Flush();
+            pageData.Flush();
```

Comment:
Have to `flush` both? Flushing the `pageWriter` should be enough.

Reply:
You're probably right, though as agreed I am removing this project from the solution as it is in a way a duplicate of AzureFunctionBasicHandler and would just cause confusion.